src/program.js

```diff
@@ -341,6 +341,15 @@ Example: $0 --help run.
         demand: false,
         type: 'string',
       },
+      'firefox-port': {
```

Comment:
This name should make it more obvious that you are connecting to a remote Firefox instance. Maybe `remote-firefox-port`.